flink-runtime-web/pom.xml

```diff
@@ -263,8 +263,8 @@ under the License.
 							<goal>install-node-and-npm</goal>
 						</goals>
 						<configuration>
-							<nodeVersion>v16.13.2</nodeVersion>
-							<npmVersion>8.1.2</npmVersion>
+							<nodeVersion>v18.19.1</nodeVersion>
```

Comment:
Maintenance of node.js v18 LTS will end in May 2025, what about using v20 LTS directly?

Reply:
> Maintenance of node.js v18 LTS will end in May 2025, what about using v20 LTS directly?

ref https://nodejs.org/en/about/previous-releases